### PR TITLE
Refresh after post creation

### PR DESCRIPTION
**Pull request by AhmedAdel999**
Forced a refresh each time the user creates a new post.

### CODE REVIEW

File: modules/shared/components/organisms/PostCreation/PostCreation.tsx [hunk 1]
```diff
@@ -159,6 +159,7 @@ const PostCreation: FC<IPostCreation.IProps> = ({
         await logoutUser();
         redirectToLoginPage();
       }
+      window.location.reload();
```
Comment (Nilegfx) [inline modules/shared/components/organisms/PostCreation/PostCreation.tsx:162]: Wohhoo wohhoo.. that's too aggressive for SPA app 😂

Comment (AhmedAdel999) [reply modules/shared/components/organisms/PostCreation/PostCreation.tsx:162]: There is another way using next router would that be better?

File: modules/shared/components/organisms/PostCreation/PostCreation.tsx [hunk 2]
```diff
@@ -159,6 +161,7 @@ const PostCreation: FC<IPostCreation.IProps> = ({
         await logoutUser();
         redirectToLoginPage();
       }
+      router.reload();
```
Comment (atwfk) [inline modules/shared/components/organisms/PostCreation/PostCreation.tsx:164]: use 
```typescript
router.replace(router.asPath) as Promise<NextRouter>
```
as if the user tried to go back to the previous page, next js will ignore the browser refresh and that's what he expects

**Review by atwfk — CHANGES_REQUESTED** [review]

**Review by atwfk — APPROVED** [review]
LGTM

**Review by Nilegfx — CHANGES_REQUESTED** [review]
this is not the desired solution. I prefer not to merge it.

**Pull request closed without merging**